api/requirements.txt

```diff
@@ -21,3 +21,5 @@ pytz==2022.1
 requests==2.28.0
 sqlparse==0.4.2
 toml==0.10.2
+psycopg2==2.8.6
+
```

Comment:
Is this required?

Reply:
I needed that for my docker setup

Reply:
I don't think this is required if you follow the docs.